services/uploads/src/lib/avScan.ts

```diff
@@ -47,7 +47,16 @@ export async function scanFile(
     }
 
     const tags = generateVirusScanTagSet(tagResult)
-    const err = await s3Client.tagObject(key, bucket, tags)
+    // tagObject replaces existing tags, so we get the tags, add the new ones, and then set them all back
+    const currentTagsResult = await s3Client.getObjectTags(key, bucket)
+    if (currentTagsResult instanceof Error) {
+        console.error('Failed to get object tags', currentTagsResult)
```

Comment:
I'll wait to hear from mojo on this one.

Reply:
So yeah, I did this intentionally -- this is the lib file that we call in the lambda handler, so I put the OTEL logging at the level of our handlers. I wanted to keep the instrumentation at the place we call it. 

We might want to refactor this a bit so that we drop using `console.error` here and just return the error though.

Reply:
Thanks.  I took them out.  Let me know if you intended anything more or less 🙂

Reply:
we might want to pass the trace down so that things can be added to it? Or we should rely on the returned error to have enough info for logging to otel.